Fallout2/Fallout1in2/mods/fo1_lang_german/text/german/dialog/MASTER.MSG

```diff
@@ -117,8 +117,8 @@
 {300}{}{Däh!}
 
 # Master Tell-Me-Abouts generic responses ((set_tma_data(NAME, 981, -1, -1, 980, 981) --BreakinBenny);
-{980}{}{Einheit}
-{981}{}{Meister}
+{880}{}{Einheit}
```

Comment:
Now then... 880 to 881 as keyword messages should work, and Master using TMA_MSG_DONTKNOW_START (980) and message 981 be fine.

Reply:
Stop using the web editor of github, you break the encoding of the files!
Fallout is very specific in non english languages...

Reply:
Got it. I'll only touch the English message files.